_posts/2023-05-03-refresh-at-trino-fest.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Refreshing at the lakehouse summer camp"
+author: "Manfred Moser, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/trino-fest.png
+---
+
+Summer is just around the corner, and we are busy getting ready for [Trino Fest
+2023]({% post_url 2023-04-05-announcing-trino-fest-2023 %}). Everything is
+ramping up. Early birds are starting to register, and [so should
+you](https://www.starburst.io/info/trinofest). Our Trino Fest theme song is
+available for your listening pleasure, and we are reviewing speaker submissions.
+The festival is promising to be another great event to learn about lakehouse use
+cases with Trino, but we are also featuring some great presentations for
+querying data with Trino. And of course, we are still looking for more
+presenters, so don't hesitate and [submit your
+proposal](https://sessionize.com/trino-fest-2023).
+
+<!--more-->
+
+Before you dive into the technical details of our upcoming conference, lean back
+and listen to our theme song. Hopefully you are feeling the summer vibe coming
+your way already.
+
+{% youtube 6oN-70jSbF8 %}
+
+Our event host [Starburst](https://www.starburst.io/) is again helping us ensure
+that Trino Fest is a venue for Trino beginners and experts to meet, exchange
+ideas, and learn from each other. One of the Starburst engineers, [Jan
+Waś](https://github.com/nineinchnick), is scheduled to present about his
+amazingly low-effort setup to use Trino for analysis and report generation.
+
+Getting closer to the theme of the event "Lakehouse summer camp", we are certain
+to have sessions about Iceberg, Delta Lake, and Hudi usage with Trino. Learn
```

Comment:
I think "we are planning on having sessions" instead of "we are certain to have sessions" sounds a little more concrete.